daprdocs/content/en/developing-applications/building-blocks/workflow/howto-manage-workflow.md

```diff
@@ -30,7 +30,19 @@ WorkflowReference startResponse = await daprClient.StartWorkflowAsync(orderId, w
 GetWorkflowResponse getResponse = await daprClient.GetWorkflowAsync(orderId, workflowComponent, workflowName);
 
 // Terminate the workflow
-await daprClient.TerminateWorkflowAsync(instanceId, workflowComponent);
+GetWorkflowResponse getResponse = await daprClient.TerminateWorkflowAsync(orderId, workflowComponent, cts);
```

Comment:
There is no response object returned from the terminate call. It should look like this:
await daprClient.TerminateWorkflowAsync(orderId, workflowComponent, cts);

This applies to the RaiseEvent, Pause, Resume, and Purge calls below as well